src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js

```diff
@@ -129,101 +145,159 @@ export class CernerCallToAction extends Component {
-            const facilityName = facility?.attributes?.name;
-            const systemName = getVamcSystemNameFromVhaId(
-              ehrDataByVhaId,
-              strippedID,
-            );
-            const isCerner = cernerFacilities?.some(
-              cernerFacility => cernerFacility?.facilityId === strippedID,
-            );
-
-            return (
-              <p className="usa-alert-text vads-u-margin-bottom--2" key={id}>
-                <strong>{systemName || facilityName}</strong>{' '}
-                {isCerner
-                  ? '(Now using My VA Health)'
-                  : '(Using My HealtheVet)'}
+          {featureStaticLandingPage && (
+            <>
+              <h3 className="usa-alert-heading">
+                Choose the right health portal
+              </h3>
+              <p className="vads-u-font-weight--bold">
+                To manage appointments at these facilities, go to My VA Health:
               </p>
-            );
-          })}
-
-          <p className="usa-alert-text">
-            Choose a health management portal, depending on your provider&apos;s
-            facility. You may need to disable your browser&apos;s pop-up blocker
-            to open the portal. If you&apos;re prompted to sign in again, use
-            the same account you used to sign in on VA.gov.
-          </p>
-
-          <h4 className="vads-u-margin-y--3">{linksHeaderText}</h4>
-
-          {/* List of user's facility links */}
-          {map(facilities, facility => {
-            // Derive facility properties.
-            const id = facility?.id;
-            const strippedID = replace(id, 'vha_', '');
-            const facilityName = facility?.attributes?.name;
-            const systemName = getVamcSystemNameFromVhaId(
-              ehrDataByVhaId,
-              strippedID,
-            );
-            const isCerner = cernerFacilities?.some(
-              cernerFacility => cernerFacility?.facilityId === strippedID,
-            );
-
-            // Derive the link text/label.
-            const linkText = isCerner
-              ? 'Go to My VA Health'
-              : myHealtheVetLinkText;
-
-            return (
-              <div key={`${id}-cta-link`}>
+              <div className="vads-u-margin-y--1">
+                <ul className="vads-u-margin-left--1p5">
+                  <ListItem
+                    facilities={facilities}
+                    ehrDataByVhaId={ehrDataByVhaId}
+                  />
+                </ul>
+                <a
+                  className="vads-c-action-link--blue"
+                  href={myVAHealthLink}
+                  onClick={onCTALinkClick}
+                  rel="noreferrer noopener"
+                  target="_blank"
+                >
+                  Go to My VA Health
+                </a>
+              </div>
+              <va-additional-info trigger="Having trouble opening My VA health? Try these steps:">
```

Comment:
"Try these steps:" should be removed from the title and repositioned above the start of item lists.